installer/volcano-development.yaml

```diff
@@ -120,6 +120,8 @@ spec:
           secret:
             defaultMode: 420
             secretName: volcano-admission-secret
+      nodeSelector:
+        kubernetes.io/arch: amd64
```

Comment:
I think we should not update this yaml which is used for x64 hosts :)

Reply:
Ok sure.